functests/updateservice_creation_test.go

```diff
@@ -172,9 +184,30 @@ func TestCustomResource(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	ctx = context.Background()
+	graph_data, _ := os.LookupEnv("GRAPH_DATA")
```

Comment:
nit: `os.LookupEnv` with ignored boolean is `os.Getenv` with more syntax clutter :)

Reply:
I've pushed f605e05 -> 68f74fc with a pivot to `Getenv`.